docker/Dockerfile.i686-pc-windows-gnu

```diff
@@ -34,12 +34,20 @@ RUN mkdir -p /usr/lib/binfmt-support/ && \
 COPY windows-entry.sh /
 ENTRYPOINT ["/windows-entry.sh"]
 
+COPY toolchain.cmake /opt/toolchain.cmake
+ENV CMAKE_TOOLCHAIN_FILE=/opt/toolchain.cmake
+ENV CROSS_CMAKE_OBJECT_FLAGS="-ffunction-sections -fdata-sections -m32"
+
 # for why we always link with pthread support, see:
 # https://github.com/cross-rs/cross/pull/1123#issuecomment-1312287148
 ENV CROSS_TOOLCHAIN_PREFIX=i686-w64-mingw32-
 ENV CROSS_SYSROOT=/usr/i686-w64-mingw32
+ENV CROSS_TARGET_RUNNER="wine"
 ENV CARGO_TARGET_I686_PC_WINDOWS_GNU_LINKER="$CROSS_TOOLCHAIN_PREFIX"gcc-posix \
-    CARGO_TARGET_I686_PC_WINDOWS_GNU_RUNNER=wine \
+    CARGO_TARGET_I686_PC_WINDOWS_GNU_RUNNER="$CROSS_TARGET_RUNNER" \
     CC_i686_pc_windows_gnu="$CROSS_TOOLCHAIN_PREFIX"gcc-posix \
     CXX_i686_pc_windows_gnu="$CROSS_TOOLCHAIN_PREFIX"g++-posix \
-    BINDGEN_EXTRA_CLANG_ARGS_i686_pc_windows_gnu="--sysroot=$CROSS_SYSROOT"
+    BINDGEN_EXTRA_CLANG_ARGS_i686_pc_windows_gnu="--sysroot=$CROSS_SYSROOT" \
+    CROSS_CMAKE_SYSTEM_NAME=Windows \
```

Comment:
For some reason, `cmake-rs` passes this as `windows` only for 32-bit Windows GNU but the correct value is `Windows`, and `Platform/windows` is unknown to CMake. So, this differs **slightly** from cmake-rs.